book/0 - Introduction/3 - History.html

```diff
@@ -1,5 +1,5 @@
 <p>
-	MCServer was created by <a href="https://github.com/faketruth">FakeTruth</a> in October of 2011 as a replacement for the vanilla server, designed to be better performing and more configurable. It was soon open-sourced and developers started working.
+	Cuberite was created in October of 2011 by <a href="https://github.com/faketruth">FakeTruth</a> with it's original name "MCServer", as a replacement for the vanilla server, designed to be better performing and more configurable. It was soon open-sourced and developers started working.
```

Comment:
I think this is wrong. In 2011 the first commit online maybe was made, but MCServer existed long before that. There is a [post by FakeTruth on the minecraftforums](http://www.minecraftforum.net/forums/archive/alpha/alpha-survival-multiplayer/824720-mcserver-a-c-server-by-one-man) that dates at December 7 2010, but it probably existed even longer before that. (since I assume Fake didn't create MCServer in one day ^^)